src/game/Tactical/Soldier_Profile.cc

```diff
@@ -608,6 +608,8 @@ SOLDIERTYPE* ChangeSoldierTeam(SOLDIERTYPE* const old_s, UINT8 const team)
 	new_s->bLastRenderVisibleValue = old_s->bLastRenderVisibleValue;
 	new_s->bVisible                = old_s->bVisible;
 	new_s->bCamo                   = old_s->bCamo;
+	new_s->bLevel                  = old_s->bLevel;
+	if(old_s->bLevel == 1) SetSoldierHeight(new_s, 58.0);
```

Comment:
Ok, can you then please make it all better an create a named constant and use it everywhere? No need to rewrite this commit, just push extra ones to the same branch.